samples/ElmPageSample/ElmPageSample.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Also, can we remove `CoreFxVersion` from `dependencies.props`? Presumably it's no longer referenced from this repo.